tests/unit/test_info.py

```diff
@@ -53,7 +53,10 @@ def test_info_in_broken_git_repo(tmp_dir, dvc, scm, caplog):
     assert "Repo: dvc, git (broken)" in dvc_info
 
 
-def test_caches(tmp_dir, dvc, caplog):
+def test_caches(tmp_dir, dvc, caplog, mocker):
+    from dvc.tree.ssh import SSHTree
+
+    mocker.patch.object(SSHTree, "exists", return_value=False)
```

Comment:
now that we will try to load a config file on remote connection, this has to be mocked out for these kinds of tests with a fake remote URL otherwise the test will hang until the network call times out